docs/wiki/00_SERVER_INSTALL_AND_CONFIGURATION/03_LORIS_MRI_Install/Signal_Viewer_UI_Install.md

```diff
@@ -0,0 +1,81 @@
+# EEG Visualization Install
+
+> Last update: 2022-11-02
+
+## Context
+
+This document is a guide to install and configure Loris to use the `EEG Visualization` features.
+
+## TOC
+
+1. Install
+    1. [Environment](#environment)
+    1. [Installation requirements](#installation-requirements)
+1. [Import EEG BIDS data to Loris](#import-eeg-bids-data-to-loris)
+
+## Install
+
+### Environment
+
+- `Loris` `v24.0` or above.
+- `Loris-MRI` installed.
+
+
+### Installation requirements
+
+Follow the [installation steps for the visualization features](https://github.com/aces/Loris/tree/main/modules/electrophysiology_browser#-installation-requirements-to-use-the-visualization-features).
+
+
+## Import EEG BIDS data to Loris
+
+Adapt and use the following `pipeline.sh` script. It will:
+- take BIDS archives located in `/data/incoming`
+- extract them to their folder in `/data/processing/bidsFiles/`
+- move the archive to `/data/archive`
+- import data into Loris DB with `/opt/$PROJECT_NAME/bin/mri/python/bids_import.py`
+
+```bash
+#!/usr/bin/env bash
+
+# To execute periodically, add
+# */15 * * * * cd /var/www/loris/project/tools && ./pipeline.sh
+# in crontab -e
+
+# change the <UPDATE_HERE> with your project name
+PROJECT_NAME='<UPDATE_HERE>'
+DATA_FOLDER='/data/'${PROJECT_NAME}
+
+source /opt/$PROJECT_NAME/bin/mri/environment
```

Comment:
if the script gets moved to LORIS-MRI, then you can remove this line since all LORIS-MRI script should be run after having sourced the environment (which is often done by default in bashrc for most people)